Kudu.Core/Tracing/KuduEvent.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kudu.Core.Tracing
+{
+    class KuduEvent
+    {
+        public string siteName = string.Empty;
+        public string projectType = string.Empty;
+        public string result = string.Empty;
+        public string error = string.Empty;
+        public string deploymentDurationInMilliseconds = string.Empty;
+        public string siteMode = string.Empty;
+        public string scmType = string.Empty;
+        public string vsProjectId = string.Empty;
+        public string jobName = string.Empty;
+        public string scriptExtension = string.Empty;
+        public string jobType = string.Empty;
+        public string trigger = string.Empty;
+        public string method = string.Empty;
+        public string path = string.Empty;
+        public string Message = string.Empty;
+        public string exception = string.Empty;
+        public string route = string.Empty;
+        public string userAgent = string.Empty;
+        public string requestId = string.Empty;
+        public string buildVersion = string.Empty;
```

Comment:
Currently, KuduLite build version is a constant in Constants.cs (KuduBuild). We can refer that directly here currently, this would be overridden with an auto incremented build when we link it with our CI pipeline.